addons/hr_recruitment/views/hr_recruitment_views.xml

```diff
@@ -25,24 +25,20 @@
         <field name="model">hr.applicant</field>
         <field name="arch" type="xml">
             <tree string="Applicants" decoration-bf="message_needaction==True" multi_edit="1">
-                <field name="message_needaction" invisible="1"/>
-                <field name="last_stage_id" invisible="1"/>
-                <field name="create_date" readonly="1"/>
-                <field name="date_last_stage_update" invisible="1"/>
-                <field name="name" readonly="1"/>
-                <field name="partner_name" readonly="1"/>
-                <field name="email_from" readonly="1"/>
-                <field name="partner_phone" widget="phone" readonly="1"/>
+                <field name="name"/>
+                <field name="partner_name"/>
                 <field name="job_id"/>
                 <field name="stage_id"/>
-                <field name="medium_id"/>
-                <field name="source_id" readonly="1"/>
-                <field name="priority"/>
-                <field name="type_id" invisible="1"/>
-                <field name="availability" invisible="1"/>
-                <field name="department_id" invisible="context.get('invisible_department', True)" readonly="1"/>
-                <field name="user_id"/>
-                <field name="company_id" groups="base.group_multi_company" readonly="1"/>
+                <field name="create_date" optional="show"/>
+                <field name="priority" optional="show"/>
+                <field name="salary_expected" optional="show"/>
+                <field name="availability" optional="show"/>
+                <field name="salary_proposed" optional="hidden"/>
+                <field name="email_from" optional="hidden"/>
+                <field name="partner_phone" optional="hidden"/>
+                <field name="medium_id" optional="hidden"/>
+                <field name="source_id" optional="hidden"/>
```

Comment:
Why did you remove the readonly modifier on some of those fields?

Reply:
When I made the new view tree I forgot to put the readonly attributes on some fields